packages/react-components/src/components/Switch/Switch.stories.tsx

```diff
@@ -1,23 +1,35 @@
 import * as React from 'react';
-import { ComponentMeta } from '@storybook/react';
+import { ComponentMeta, Story } from '@storybook/react';
 
-import { Switch as SwitchComponent, SwitchProps } from './Switch';
-import noop from '../../utils/noop';
+import { Switch, SwitchProps } from './Switch';
 
 export default {
   title: 'Components/Switch',
-  component: SwitchComponent,
-} as ComponentMeta<typeof SwitchComponent>;
-
-export const Switch = (args: SwitchProps): React.ReactElement => {
-  return (
-    <div>
-      <SwitchComponent {...args} />
-    </div>
-  );
-};
+  component: Switch,
+} as ComponentMeta<typeof Switch>;
+
+const Template: Story<SwitchProps> = (args: SwitchProps) => (
+  <Switch {...args} />
+);
 
-Switch.args = {
-  size: 'basic',
-  onChange: noop,
+const basicWithoutStateHandler: Partial<SwitchProps> = {
+  onChange: undefined,
 };
+
+export const BasicSize = Template.bind({});
+BasicSize.args = { ...basicWithoutStateHandler, size: 'basic' };
+
+export const CompactSize = Template.bind({});
+CompactSize.args = { ...basicWithoutStateHandler, size: 'compact' };
+
+export const EnabledOn = Template.bind({});
+EnabledOn.args = { ...basicWithoutStateHandler, on: true };
+
+export const EnabledOff = Template.bind({});
+EnabledOff.args = { ...basicWithoutStateHandler, on: false };
+
+export const DisabledOn = Template.bind({});
+DisabledOn.args = { ...basicWithoutStateHandler, on: true, disabled: true };
+
+export const DisabledOff = Template.bind({});
+DisabledOff.args = { ...basicWithoutStateHandler, on: false, disabled: true };
```

Comment:
I think we can minimize the number of different stories here. We can divide it into "kinds" and "states", just like we have in `Button` component stories.

Reply:
Addressed in https://github.com/livechat/design-system/pull/353/files/b1ba318006993158ba0bbe83499b5172ff99d8fd..7b2f4ecdd7189efd1a793d7d4fc2bbda4339a3a0

Reply:
@marcinsawicki Please check if this is what you've expected https://github.com/livechat/design-system/pull/353/commits/e443d1288ea431e9ba211fd8e717612c0ad8c826

Reply:
It's perfect 👍